src/Workspaces/Core/Portable/Formatting/Engine/TokenStream.OriginalTriviaDataGetter.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Formatting
+{
+    internal partial class TokenStream
+    {
+        /// <summary>
+        /// Retrieves original <see cref="TriviaData"/> via <see cref="TokenStream.GetOriginalTriviaData(TokenData, TokenData)"/>
+        /// </summary>
+        private struct OriginalTriviaDataGetter : ITriviaDataGetter
+        {
+            private readonly TokenStream _tokenStream;
+            public OriginalTriviaDataGetter(TokenStream tokenStream)
```

Comment:
nit: we put a blank line between fields and constructor.  (`nit` means "minor".  but i still call out so you can get a feel for the codestyle of the codebase).